terkin/device.py

```diff
@@ -243,10 +243,8 @@ def add(item=''):
         add('=' * len(title))
 
         # Machine runtime information.
-        if self.platform_info.vendor == self.platform_info.MICROPYTHON.Pycom:
-            frequency = machine.freq() / 1000000
-        else:
-            frequency = machine.freq()[0] / 1000000
+        frequency = machine.freq() / 1000000
+
```

Comment:
I see. Is it actually just the same thing on both platforms?

Maybe it will be different on the pyboard-D. We will see that when I will try to ramp up the code again on this platform.

Reply:
Yes, surprised me, too. Maybe I had one weird firmware version.